launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -67,9 +67,17 @@ void CSettingsView::loadSettings()
 	int resIndex = ui->comboBoxResolution->findText(QString("%1x%2").arg(resX).arg(resY));
 
 	ui->comboBoxResolution->setCurrentIndex(resIndex);
-	ui->comboBoxFullScreen->setCurrentIndex(settings["video"]["fullscreen"].Bool());
 	ui->comboBoxShowIntro->setCurrentIndex(settings["video"]["showIntro"].Bool());
+
+#ifdef Q_OS_IOS
+    ui->comboBoxFullScreen->setCurrentIndex(true);
+    ui->checkBoxFullScreen->setChecked(false);
+    for (auto widget : std::initializer_list<QWidget *>{ui->comboBoxFullScreen, ui->checkBoxFullScreen})
+        widget->setDisabled(true);
+#else
+    ui->comboBoxFullScreen->setCurrentIndex(settings["video"]["fullscreen"].Bool());
```

Comment:
From coding guidelines
Indentation
Use tabs for indentation. If you are modifying someone else's code, try to keep the coding style similar.

Reply:
thanks, missed to format that one